text/0000-oci-image-annotations.md

```diff
@@ -0,0 +1,116 @@
+
+The outcome will be that users and tools will be able to gather more information about buildpacks, facilitating use cases such as gathering releases notes and finding documentation.
+
+# What it is
+[what-it-is]: #what-it-is
+
+`pack buildpack package` should set the following OCI annotations on the images it produces:
+
+- `org.opencontainers.image.source` (when possible)
+- `org.opencontainers.image.revision` (when possible)
+- `org.opencontainers.image.title`
+- `org.opencontainers.image.version`
+- `org.opencontainers.image.url` (when possible)
+- `org.opencontainers.image.description` (when possible)
+
+The target personas as buildpack users, platform operators, and platform implementers. Any of those groups will be able to more easily understand the origin (source), version, and other information about the buildpack. This information can then be used manually or with the aid of tools to get release notes which aid these personas in making informed decisions.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+When building the buildpack, the `pack` tool can get the values for the `org.opencontainers.image.source` and `org.opencontainers.image.revision` annotations from git. `org.opencontainers.image.source` is derived from the git origin and `org.opencontainers.image.revision` is the git commit hash.
+
+The other annotation values come from `buildpack.toml` mapped to OCI annotations as follows:
+
+- `name` -> `org.opencontainers.image.title`
+- `version` -> `org.opencontainers.image.version`
+- `homepage` (optional) -> `org.opencontainers.image.url`
+- `description` (optional) -> `org.opencontainers.image.description`
+
+The following example values are from [Paketo Buildpack for Java 13.0.1](https://github.com/paketo-buildpacks/java/releases/tag/v13.0.1):
+
+- `org.opencontainers.image.source`: https://github.com/paketo-buildpacks/java
+- `org.opencontainers.image.revision`: 09747b1df0a56aea74ce9b01af89df6feb1fc50a
+- `org.opencontainers.image.title`: Paketo Buildpack for Java
+- `org.opencontainers.image.version`: 13.0.1
+- `org.opencontainers.image.url`: https://paketo.io/docs/howto/java
+- `org.opencontainers.image.description`: A Cloud Native Buildpack with an order definition suitable for Java applications
+
+# Migration
+[migration]: #migration
+
+The `pack` tool would be modified to set the annotations. Because the tool _should_ set these annotations (not _must_ set), buildpacks created with earlier versions of the tool are still consider to be valid in accordance with the distribution specification.
```

Comment:
```suggestion
The `pack` tool would be modified to set the annotations. Because the tool _should_ set these annotations (not _must_ set), buildpacks created with earlier versions of the tool are still considered to be valid in accordance with the distribution specification.
```